x-pack/elastic-agent/_meta/elastic-agent.yml

```diff
@@ -9,6 +9,26 @@ outputs:
     password: changeme
 
 inputs:
+  - type: synthetics/monitors
```

Comment:
Is there any common configuration that would go into `synthetics/monitors`? If not, we just also go with:


```
  - type: synthetics/http
    name: MyHTTPMonitor
    id: my-id
    target: "https://elastic.co"
    schedule: "@every 10s"
    http: # example HTTP specific option
      user: blah
      password: something else
  - type: synthetics/icmp
    name: MyICMPMonitor
    id: my-icmp-id
    target: "elastic.co"
    schedule: "@every 10s"
  - type: synthetics/tcp
    name: MyTCPMonitor
    id: my-tcp-id
    target: "elastic.co:80"
    schedule: "@every 10s"
```